web/src/main/java/ua/com/vertex/controllers/CertificateDetailsPageController.java

```diff
@@ -29,50 +28,31 @@ public String showCertificateDetailsPage() {
     }
 
     @RequestMapping(value = "/getCertificate")
```

Comment:
No, I can't. There are three different requests inside this controller. The implementation of @GetMapping, @PostMapping, @PutMapping, etc. is possible if the request is the same for all of them.